etc/kayobe/ansible/ceph-config.yml

```diff
@@ -21,69 +21,72 @@
       - { name: "glance", keyring: "ceph.client.glance.keyring" }
       - { name: "nova", keyring: "ceph.client.cinder.keyring" }
   tasks:
-    - block:
-        # Create pools and users.
-        - name: Create and initialise pools for OpenStack services
-          command:
-            cmd: >
-              cephadm shell --
-              ceph osd pool create {{ item }}
-          with_items: "{{ cephadm_ceph_pools }}"
-          become: true
+    # Create pools and users.
+    - name: Create and initialise pools for OpenStack services
+      command:
+        cmd: >
+          cephadm shell --
+          ceph osd pool create {{ item }}
+      with_items: "{{ cephadm_ceph_pools }}"
+      become: true
 
-        - name: Create users for OpenStack services
-          command:
-            cmd: >
-              cephadm shell --
-              ceph auth get-or-create {{ item }}
-          become: true
-          with_items: "{{ cephadm_ceph_users }}"
+    - name: Create users for OpenStack services
+      command:
+        cmd: >
+          cephadm shell --
+          ceph auth get-or-create {{ item }}
+      become: true
+      with_items: "{{ cephadm_ceph_users }}"
 
-        - name: Check ceph health
-          command:
-            cmd: cephadm shell -- ceph health detail
-          become: True
-          changed_when: false
+    - name: Check ceph health
+      command:
+        cmd: cephadm shell -- ceph health detail
+      become: True
+      changed_when: false
 
-        # Create config files.
-        - name: Ensure required kolla config directories exist
-          file:
-            state: directory
-            name: "{{ kayobe_config_path }}/kolla/config/{{ item.name }}"
-            mode: 0755
-          with_items: "{{ cephadm_kolla_ceph_services }}"
-          delegate_to: localhost
+    - name: Create ceph.conf
+      command:
+        cmd: cephadm shell -- cat /etc/ceph/ceph.conf | tee /etc/ceph/ceph.conf
```

Comment:
Why not use fetch?

Reply:
I don't think we can use fetch, as the file is not present on the host. I think it is generated on the fly by cephadm?

Reply:
Of course, it executes on the remote hosts. Do you need to use the shell module to pipe?

Reply:
I used cephadm again and this time it created `ceph.conf` on each host. I am removing this part.